tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -25,3 +25,23 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: sharee search files using content
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has uploaded file with content "hello world" to "uploadFolder/file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "uploadFolder/file2.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "uploadFolder/file3.txt"
```

Comment:
Refer to https://github.com/owncloud/ocis/pull/6709#discussion_r1266270454

Reply:
Ok